src/resources/Enums.ts

```diff
@@ -1053,3 +1053,13 @@ export enum FieldValueType {
     ARRAY = 'ARRAY',
     HIERARCHIC_MULTI_VALUE = 'HIERARCHIC_MULTI_VALUE',
 }
+
+export enum ProductsSortCriteria {
+    alphanumeric = 'alphanumeric',
+    score = 'score',
+}
```

Comment:
Server side, we call those `FacetRequestSortType`. I think we should have `facet` in the name.